components/title/basic/src/index.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-basic-main-title: $c-primary !default;
+$c-title-basic-subtitle: $c-primary !default;
```

Comment:
Yes, I think it will be better to override this var $c-title-basic-main-title in each theme-basic, but with these definitions we ensure to have at least a default value.